app/views/users/edit.html.slim

```diff
@@ -1,4 +1,9 @@
-h2 = t('headings.users.edit', name: @user.email)
+h2 = t('headings.users.edit')
+
+ul.no-bullets
+  li = t('user.email_html', attr: @user.email)
```

Comment:
let's do it separately, please! PRs separated by functional changes are VERY helpful when looking through git history